application-services/custom/influxdb-export/pkg/transforms/converions.go

```diff
@@ -0,0 +1,70 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//
+// Copyright (C) 2020 IOTech Systems LTD
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package transforms
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/appcontext"
+	"github.com/edgexfoundry/go-mod-core-contracts/models"
+)
+
+type Conversion struct {
+}
+
+func NewConversion() Conversion {
+	return Conversion{}
+}
+
+func (f Conversion) TransformToInflux(edgexcontext *appcontext.Context, params ...interface{}) (continuePipeline bool, stringType interface{}) {
+	if len(params) < 1 {
+		return false, errors.New("No Event Received")
+	}
+
+	edgexcontext.LoggingClient.Debug("Transforming to InfluxDB Line Protocol format")
+
+	event, ok := params[0].(models.Event)
+
+	if !ok {
+		return false, errors.New("Unexpected type received")
+	}
+
+	var buffer strings.Builder
+
+	// write device name as measurement
+	buffer.WriteString(event.Device)
+	// write tags if any, comma separated [,tag<index>=<tag_value>[,tag<index>=<tag_value>]]
+	for i, tag := range event.Tags {
+		// write comma
+		buffer.WriteString(",")
+		buffer.WriteString("tag")
+		buffer.WriteString(i)
+		buffer.WriteString("=")
+		buffer.WriteString(tag)
+	}
+	// write space
+	buffer.WriteString(" ")
+	// write fields (readings) comma separated <field_key>=<field_value>[,<field_key>=<field_value>]
```

Comment:
This comment still doesn't match what the code is producing, i.e. code doesn't add no square brackets
Should be:
```
write fields (readings) comma separated <field_key>=<field_value>,<field_key>=<field_value>
```

Reply:
square brackets just indicate that each subsequent field is optional (after the first).
Again, using the syntax of Influx
<measurement>[,<tag_key>=<tag_value>[,<tag_key>=<tag_value>]] **<field_key>=<field_value>[,<field_key>=<field_value>]** [<timestamp>]

Reply:
fixed